packages/account/src/Containers/employment-tax-details-container/employment-tax-details-container.tsx

```diff
@@ -0,0 +1,211 @@
+                                    tax_identification_confirm: false,
+                                },
+                                true
+                            );
+                        }}
+                        value={values.tin_skipped}
+                        label={localize('I do not have tax information')}
+                        withTabIndex={0}
+                        data-testid='tin_skipped'
+                        label_font_size={!isDesktop ? 'xxs' : 'xs'}
+                        label_line_height='m'
+                    />
+                )}
+                <div ref={tax_residence_ref} className='account-form__fieldset'>
+                    <TaxResidenceField
+                        disabled={isFieldDisabled('tax_residence')}
+                        is_tax_residence_popover_open={is_tax_residence_popover_open}
+                        setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                        setIsTinPopoverOpen={setIsTinPopoverOpen}
+                        required={(should_display_long_message && !values.tin_skipped) || is_tin_required}
+                    />
+                </div>
+                <div ref={tin_ref} className='account-form__fieldset'>
+                    <TaxIdentificationNumberField
+                        disabled={isFieldDisabled('tax_identification_number') || values.tin_skipped}
+                        is_tin_popover_open={is_tin_popover_open}
+                        setIsTinPopoverOpen={setIsTinPopoverOpen}
+                        setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                        required={(should_display_long_message && !values.tin_skipped) || is_tin_required}
+                    />
+                </div>
+                {should_show_tax_confirm_checkbox && (
+                    <Checkbox
+                        name='tax_identification_confirm'
+                        className='employment_tax_detail_field-checkbox'
+                        data-lpignore
+                        onChange={() =>
+                            setFieldValue('tax_identification_confirm', !values.tax_identification_confirm, true)
+                        }
+                        value={values.tax_identification_confirm}
+                        label={
+                            should_display_long_message
+                                ? localize(
+                                      'I hereby confirm that the tax information provided is true and complete. I will also inform {{legal_entity_name}} about any changes to this information.',
+                                      {
+                                          legal_entity_name: getLegalEntityName('maltainvest'),
+                                      }
+                                  )
+                                : localize('I confirm that my tax information is accurate and complete.')
+                        }
```

Comment:
Why don't we use the component <Localize /> here 🙏

Reply:
We can use. I'll change this